apps/condo/domains/common/components/containers/GlobalStyle.tsx

```diff
@@ -63,27 +63,45 @@ export default function GlobalStyle () {
                   position: relative;
                   bottom: 4px;
                 }
+                
+                & .ant-checkbox-inner {
```

Comment:
it's better to move these styles inside the component and use our own customized component rather than change the global

Reply:
Agree

Reply:
+1. We'll try to remove global styles soon :)